kong/cmd/utils/prefix_handler.lua

```diff
@@ -168,6 +184,16 @@ local function compile_conf(kong_config, conf_template)
     end
   end
 
+  if type(kong_config.lua_ssl_trusted_certificate) == "table" then
+    if #kong_config.lua_ssl_trusted_certificate == 0 then
+      kong_config.lua_ssl_trusted_certificate = nil
+
+    elseif #kong_config.lua_ssl_trusted_certificate == 1 then
+      kong_config.lua_ssl_trusted_certificate = kong_config.lua_ssl_trusted_certificate[1]
```

Comment:
Added this for full backards-compatibility: if only one item is used, don't even create the "hidden" file.